### PR TITLE
Polish Translation

### PR DESCRIPTION
**Pull request by deviljin112**

### CODE REVIEW

File: locales/pl.lua [hunk 1]
```diff
@@ -0,0 +1,4 @@
+Locales['en'] = {
```
Comment (rex2630) [inline locales/pl.lua]: there should be pl, not en!

**Comment by rex2630** [comment]
Please change it to fxmanifest and it should be fine

**Comment by Dislaik** [comment]
@deviljin112 My lord, change the manifest to `fxmanifest.lua`

**Comment by BrakingESX247** [comment]
I just merged @userMacieG 's polish translation 🤣 , I think that's not needed anymore.

**Pull request closed without merging**